jrcc-access-spring-boot-autoconfigure/src/main/java/ca/bc/gov/open/jrccaccess/autoconfigure/services/DocumentReadyHandler.java

```diff
@@ -47,6 +49,7 @@ public void handle(String message, TransactionInfo transactionInfo) throws Docum
 
 		if(transactionInfo == null)
 			throw new IllegalArgumentException("TransactionInfo is required.");
+		MDC.put(Constants.MDC_KEY_TRANSACTION_ID, transactionInfo.getID().toString());
```

Comment:
Are we assuming that MDC.clear will be call in the client code?

Reply:
yes.
For each client, we add filename to MDC, then invoke documentReadyHandler, then MDC.clear.
do u want to move mdc.clear to inside handler?

Reply:
Not at this time, but something to keep in mind